articles/stream-analytics/stream-analytics-with-azure-functions.md

```diff
@@ -13,7 +13,7 @@ ms.date: 01/27/2020
 
 # Tutorial: Run Azure Functions from Azure Stream Analytics jobs
 
-You can run Azure Functions from Azure Stream Analytics by configuring Functions as one of the output sinks to the Stream Analytics job. Functions are an event-driven, compute-on-demand experience that lets you implement code that is triggered by events occurring in Azure or third-party services. This ability of Functions to respond to triggers makes it a natural output to Stream Analytics jobs.
+You can run Azure Functions from Azure Stream Analytics by configuring Functions as one of the output sync's to the Stream Analytics job. Functions are an event-driven, compute-on-demand experience that lets you implement code that is triggered by events occurring in Azure or third-party services. This ability of Functions to respond to triggers makes it a natural output to Stream Analytics jobs.
```

Comment:
```suggestion
You can run Azure Functions from Azure Stream Analytics by configuring Functions as one of the output syncs to the Stream Analytics job. Functions are an event-driven, compute-on-demand experience that lets you implement code that is triggered by events occurring in Azure or third-party services. This ability of Functions to respond to triggers makes it a natural output to Stream Analytics jobs.
```

Reply:
@ShannonLeavitt , in this context sink is the right word. Outputs or the destination of processed events from a stream analytics jobs are referred to as sinks when describing stream analytics architectures/scenarios